### DISCUSSION TITLE
How to log in to your account via the mobile version of git

### DISCUSSION THREAD

**Discussion by ghost** [community/community#8186] [Mobile]
![158A3236-DB7E-485A-847F-8A02778E0D72](https://user-images.githubusercontent.com/87537312/143732375-76f7ebfc-61aa-4afc-bfa5-9414cdebb562.png)
![225AD216-EAFE-4C1C-977B-E7015A51092E](https://user-images.githubusercontent.com/87537312/143732376-775a42d4-7eab-4865-a549-b0d38d3ac287.png)
![3827F24B-4106-4B45-B9DE-FEF12F8A1595](https://user-images.githubusercontent.com/87537312/143732377-2adb8973-0a99-4b82-88fa-406742e65b97.png)

How to log in to your account via the mobile version of git, via github.LyudvigVunov.com do not log in.

**Comment by brianlovin** [comment]
Hey @LyudvigVunov I can't quite read the error messages on those views, are you able to help translate what the errors say and we can try to debug this?

**Reply by ludvigruno** [reply]
Hi, but I don't see the error.

**Reply by ludvigruno** [reply]
Let's try to debug

**Reply by brianlovin** [reply]
What does the Safari empty state view say?
